spec/models/ems_event_spec.rb

```diff
@@ -127,29 +127,17 @@
           messaging_client = double("ManageIQ::Messaging")
 
           expected_queue_payload = {
-            :service => "manageiq.ems-events",
+            :service => "ems",
```

Comment:
Why the name change?  Also, if we are sharing a kafka with another system, do we want something to effectively namespace the queue name?

Reply:
`"manageiq.ems-events"` was the name of the queue that we syndicate events to (didn't want to change that for backwards compatibility if anyone is actually using it), and `ems` is the queue name in the miq_queue table.  Prefixing with `manageiq.#{worker_queue_name}` would be general and namespace on a shared broker though

Reply:
Okay changed this to `manageiq.#{worker_queue_name}` so the topic will be namespaced